### PR TITLE
Translated current version into German

### PR DESCRIPTION
**Pull request by Seelenoede**
Removed text-about, so this section is displayed in English. Currently there is no changelog.
I decided against translating it because the translator needs to translate the changelog everytime the app gets updated.
Also I reformated the structure to the original structure.

### CODE REVIEW

**Comment by reneklacan** [comment]
Nice! Thank you very much! ... @philippt mind reviewing this?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -1,48 +1,97 @@
...
 	<string name="action_settings">Einstellungen</string>
-	<string name="action_sign_in_short">Login</string>
-	<string name="authentication_failed">Bitte versuche es noch einmal!</string>
-	<string name="button_new_conversation_discription">Neue Unterhaltung</string>
-	<string name="button_send_description">Nachricht senden</string>
-	<string name="error_apicall_failed">Verbindung fehlgeschlagen. Bitte versuche es später noch einmal</string>
-	<string name="error_incorrect_password">Dieses Passwort ist falsch</string>
-	<string name="error_invalid_password">Dieses Passwort ist ungültig</string>
-	<string name="error_invalid_username">Dieser Benutzername ist ungültig</string>
+  <string name="snakckbar_loading">Wiederhole...</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `Lade...`

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -1,48 +1,97 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<string name="app_name">FetLife</string>
+  <!-- Strings related to login -->
+  <string name="prompt_username">Benutzername</string>
+  <string name="prompt_password">Passwort</string>
+  <string name="prompt_new_message">Deine Nachricht</string>
+  <string name="action_sign_in_short">Login</string>
+  <string name="error_invalid_username">Dieser Benutzername ist ungültig</string>
+  <string name="error_invalid_password">Dieses Passwort ist ungültig</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: in fetlife-web, the message is always "invalid login" or something similar, no matter if username or password is the problem (this avoids giving people who are trying to guess passwords or nicknames more information than needed)

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -1,48 +1,96 @@
...
+  <string name="settings_title_notification_friendrequests_enabled">Benachrichtigungen bei Freundschaftsanfragen</string>
+  <string name="settings_summary_notification_friendrequests_enabled">Aktiviere Benachrichtigungen bei einkommenden Freundschaftsanfragen</string>
+  <string name="settings_title_notification_others_enabled">Andere Benachrichtigungen</string>
+  <string name="settings_summary_notification_others_enabled">Aktiviere Benachrichtigungen für alle anderen Gegebenheiten</string>
+  <string name="menu_settings">Einstellungen</string>
+  <string name="title_activity_settings">Einstellungen</string>
+  <string name="settings_title_notification_info_enabled">Info-Benachrichtigungen</string>
+  <string name="settings_summary_notification_info_enabled">Aktiviere Benachrichtigungen für allgemeine Informationen</string>
+  <string name="noification_title_new_one_or_more_notification">Neue Benachrichtigung</string>
+  <string name="noification_text_new_one_or_more_notification">Auf dich wartet eine oder mehr Benachrichtigungen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `warten`?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -1,48 +1,96 @@
...
+  <string name="settings_category_notification_text">Benachrichtigungstext</string>
+  <string name="settings_title_notification_anonymtext">Anonyme Benachrichtigungen</string>
+  <string name="settings_summary_notification_anonymtext">Verstecke jede Details in ankommenden Benachrichtigungen und nutze stattdessen eine einzige Standardbenachrichtigung</string>
+  <string name="settings_category_notification_enablement">Aktiviere Benachrichtigungen</string>
+  <string name="settings_title_notification_messages_enabled">Nachricht-Benachrichtigungen</string>
+  <string name="settings_summary_notification_messages_enabled">Aktiviere Benachrichtigungen für einkommende Nachrichten</string>
+  <string name="settings_title_notification_friendrequests_enabled">Benachrichtigungen bei Freundschaftsanfragen</string>
+  <string name="settings_summary_notification_friendrequests_enabled">Aktiviere Benachrichtigungen bei einkommenden Freundschaftsanfragen</string>
+  <string name="settings_title_notification_others_enabled">Andere Benachrichtigungen</string>
+  <string name="settings_summary_notification_others_enabled">Aktiviere Benachrichtigungen für alle anderen Gegebenheiten</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `... alle anderen Ereignisse`

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -1,48 +1,96 @@
...
+  <string name="settings_title_notifications">Benachrichtigungseinstellungen</string>
+  <string name="settings_summary_notifications">Stelle ein wann und wie Benachrichtigungen empfangen werden</string>
+  <string name="settings_category_notification_text">Benachrichtigungstext</string>
+  <string name="settings_title_notification_anonymtext">Anonyme Benachrichtigungen</string>
+  <string name="settings_summary_notification_anonymtext">Verstecke jede Details in ankommenden Benachrichtigungen und nutze stattdessen eine einzige Standardbenachrichtigung</string>
+  <string name="settings_category_notification_enablement">Aktiviere Benachrichtigungen</string>
+  <string name="settings_title_notification_messages_enabled">Nachricht-Benachrichtigungen</string>
+  <string name="settings_summary_notification_messages_enabled">Aktiviere Benachrichtigungen für einkommende Nachrichten</string>
+  <string name="settings_title_notification_friendrequests_enabled">Benachrichtigungen bei Freundschaftsanfragen</string>
+  <string name="settings_summary_notification_friendrequests_enabled">Aktiviere Benachrichtigungen bei einkommenden Freundschaftsanfragen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `eingehenden`?

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -1,48 +1,96 @@
...
+  <!-- TODO: consider translating text_about -->
+
+  <string name="settings_title_notifications">Benachrichtigungseinstellungen</string>
+  <string name="settings_summary_notifications">Stelle ein wann und wie Benachrichtigungen empfangen werden</string>
+  <string name="settings_category_notification_text">Benachrichtigungstext</string>
+  <string name="settings_title_notification_anonymtext">Anonyme Benachrichtigungen</string>
+  <string name="settings_summary_notification_anonymtext">Verstecke jede Details in ankommenden Benachrichtigungen und nutze stattdessen eine einzige Standardbenachrichtigung</string>
+  <string name="settings_category_notification_enablement">Aktiviere Benachrichtigungen</string>
+  <string name="settings_title_notification_messages_enabled">Nachricht-Benachrichtigungen</string>
+  <string name="settings_summary_notification_messages_enabled">Aktiviere Benachrichtigungen für einkommende Nachrichten</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `eingehende`?

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: This line is the same as it is in the current version.
There is also a difference in the English strings.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: Is corrected, thanks.

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -1,48 +1,96 @@
...
+  <string name="button_media_picture_upload_confirmation">Bestätigen</string>
+  <string name="button_media_picture_upload_cancel">Abbrechen</string>
+  <string name="title_media_picture_upload_confirmation">Füge dein Bild hinzu</string>
+  <!-- TODO: consider translating text_about -->
+
+  <string name="settings_title_notifications">Benachrichtigungseinstellungen</string>
+  <string name="settings_summary_notifications">Stelle ein wann und wie Benachrichtigungen empfangen werden</string>
+  <string name="settings_category_notification_text">Benachrichtigungstext</string>
+  <string name="settings_title_notification_anonymtext">Anonyme Benachrichtigungen</string>
+  <string name="settings_summary_notification_anonymtext">Verstecke jede Details in ankommenden Benachrichtigungen und nutze stattdessen eine einzige Standardbenachrichtigung</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `Verstecke Details in eingehenden Benachrichtigungen...`

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 8]
```diff
@@ -1,48 +1,96 @@
...
+  <string name="message_android_version_not_sufficient">Dieses Feature benötigt eine neuere Android-Version.</string>
+  <string name="header_friendrequest">Freundschaftsanfragen</string>
+  <string name="header_friendsuggestion">Geteilte Profile</string>
+  <string name="nfc_tap_your_device">Lege dein Gerät an ein anderes Android-Gerät, um den Beam zu aktivieren</string>
+  <string name="message_nfc_turned_off">Bitte NFC aktivieren um dieses Feature zu nutzen</string>
+  <string name="title_activity_about">Über diese App</string>
+  <string name="snackbar_version_notification">Im \'Über\'-Fenster kannst du sehen, was neu in v%1$s ist</string>
+  <string name="text_notificationhistoryitem_toremove">Lösche</string>
+  <string name="text_notificationhistoryitem_removed">Gelöscht</string>
+  <string name="notificationhistory_activity_hint">Wische nach link, um Benachrichtigungen zu löschen</string>
```
Comment (rakvat) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: `links`

**Comment by rakvat** [comment]
:+1: (some suggestions)

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 3 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: Of course. I change it.

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 4 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I think "Gegebenheiten" matches the tone in the app better than "Ereignisse".

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 8 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I'll change it

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 7 above)
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: Yeah, eingehend sounds better. I also reminds me on Stargate "eingehendes Wurmloch" :)

**Comment by Seelenoede** [comment]
Thanks for the suggestions.

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 9]
```diff
@@ -1,48 +1,96 @@
...
+  <string name="settings_category_notification_text">Benachrichtigungstext</string>
+  <string name="settings_title_notification_anonymtext">Anonyme Benachrichtigungen</string>
+  <string name="settings_summary_notification_anonymtext">Verstecke jede Details in eingehenden Benachrichtigungen und nutze stattdessen eine einzige Standardbenachrichtigung</string>
+  <string name="settings_category_notification_enablement">Aktiviere Benachrichtigungen</string>
+  <string name="settings_title_notification_messages_enabled">Nachricht-Benachrichtigungen</string>
+  <string name="settings_summary_notification_messages_enabled">Aktiviere Benachrichtigungen für eingehende Nachrichten</string>
+  <string name="settings_title_notification_friendrequests_enabled">Benachrichtigungen bei Freundschaftsanfragen</string>
+  <string name="settings_summary_notification_friendrequests_enabled">Aktiviere Benachrichtigungen bei eingehenden Freundschaftsanfragen</string>
+  <string name="settings_title_notification_others_enabled">Andere Benachrichtigungen</string>
+  <string name="settings_summary_notification_others_enabled">Aktiviere Benachrichtigungen für alle anderen Gegebenheiten</string>
```
Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: Suggestion by @rakvat - Change "Gegegebenheiten" to "Ereignisse".

File: FetLife/fetlife/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (kistitan) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I will change this also in the English version.
Can you please add the proper German text in advance?

Thanks!

Comment (Seelenoede) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: I don't have an IDE right know. Where are the "error_invalid_..." strings used?
During login I only get the "error_incorrect_password" string, no matter if username exists or not.

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 10]
```diff
@@ -1,48 +1,96 @@
...
+  <string name="notificationhistory_activity_hint">Wische nach links, um Benachrichtigungen zu löschen</string>
+  <string name="title_activity_notificationhistory">Benachrichtigungen</string>
+  <string name="menu_notifications">Benachrichtigungen</string>
+  <string name="menu_upload_media">Lade Foto hoch</string>
+  <string name="title_intent_choose_media_upload">Wähle Medien-App</string>
+  <string name="message_image_upload_started">Beame dein Foto hoch...</string>
+  <string name="message_image_upload_finished">Dein Foto wurde hochgeladen</string>
+  <string name="message_image_upload_failed">Hochladen des Fotos fehlgeschlagen. Versuche es später nochmal</string>
+  <string name="action_clear_notification_history">Lösche alle</string>
+  <string name="message_media_picture_upload_confirmation">Ich bezeuge, dass dieses Foto von mir ist, oder dass ich die expliziete Berechtigung habe, es auf Fetlife zu teilen. Ich erzähle keine Lügen</string>
```
Comment (philippt) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: explizite

File: FetLife/fetlife/src/main/res/values-de/strings.xml [hunk 11]
```diff
@@ -1,48 +1,96 @@
...
+  <string name="button_media_picture_upload_confirmation">Bestätigen</string>
+  <string name="button_media_picture_upload_cancel">Abbrechen</string>
+  <string name="title_media_picture_upload_confirmation">Füge dein Bild hinzu</string>
+  <!-- TODO: consider translating text_about -->
+
+  <string name="settings_title_notifications">Benachrichtigungseinstellungen</string>
+  <string name="settings_summary_notifications">Stelle ein wann und wie Benachrichtigungen empfangen werden</string>
+  <string name="settings_category_notification_text">Benachrichtigungstext</string>
+  <string name="settings_title_notification_anonymtext">Anonyme Benachrichtigungen</string>
+  <string name="settings_summary_notification_anonymtext">Verstecke jede Details in eingehenden Benachrichtigungen und nutze stattdessen eine einzige Standardbenachrichtigung</string>
```
Comment (philippt) [inline FetLife/fetlife/src/main/res/values-de/strings.xml]: "alle Details"?

**Comment by philippt** [comment]
:+1: Dankeschön!

**Comment by johnbaku** [comment]
We really need to start a German office... I mean.. nobody would show up to the office because we are all anti-social hermits... but never the less love the Germans!

Ich bin John!

**Comment by philippt** [comment]
> because we are all anti-social hermits...

...no idea at all who you could be talking about ;-)

**Pull request merged**